src/child/containers/sidebars/search/Search.js

```diff
@@ -38,6 +38,15 @@ class Search extends Component {
 
     onIconClick(stockCode) {
         this.props.dispatch(toggleFavourite(stockCode));
+        if (this.props.selection.code === stockCode) {
```

Comment:
This looks identical to the implementation in the Favourites component - can the duplication be removed/reduced?